test/python/ph/test_baseline.py

```
@@ -0,0 +1,118 @@
+import pytest
+
+torch = pytest.importorskip("torch")
+
+from mqt.qmap.ph.baseline import embed_target_unitary_into_chip, get_baseline_active_cols, get_baseline_input_ports
+
+
+class TestGetBaselineActiveCols:
+    """Tests for get_baseline_active_cols."""
+
+    @staticmethod
+    def test_dim2_returns_only_zero():
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `test_dim2_returns_only_zero`

Add return type annotation: `None`

(ANN205)